src/components/PlantPage.js

```diff
@@ -1,16 +1,50 @@
-import React from "react";
-import NewPlantForm from "./NewPlantForm";
-import PlantList from "./PlantList";
-import Search from "./Search";
+import React, { useState, useEffect } from "react"
+import NewPlantForm from "./NewPlantForm"
+import PlantList from "./PlantList"
+import Search from "./Search"
+import { getPlants, createPlant } from "../utils/fetchers"
 
 function PlantPage() {
+  const [plants, setPlants] = useState([])
+  const [search, setSearch] = useState("")
+
+  useEffect(() => {
+    getPlants().then((plantList) => setPlants(plantList))
```

Comment:
Just an alternate syntax option: 
```suggestion
    getPlants().then(setPlants)
```